libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Steps/BeginDialog.cs

```diff
@@ -18,6 +18,22 @@ namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Steps
     /// </summary>
     public class BeginDialog : BaseInvokeDialog
     {
+        /// <summary>
+        /// Property which is bidirectional property for input and output.  Example: user.age will be passed in, and user.age will be set when the dialog completes
+        /// </summary>
+        public string Property
+        {
+            get
+            {
+                return OutputBinding;
+            }
+            set
+            {
+                InputBindings["value"] = value;
```

Comment:
I'm prepping a separate change in C# and TypeScript for moving the constants of standard memory paths to a separate location, for now just using this pattern that is all over the code.